libs/ledger-live-common/src/families/bitcoin/wallet-btc/__tests__/wallet.errorpropagation.test.ts

```diff
@@ -26,9 +26,7 @@ describe("testing resilience of failures", () => {
             currency: "bitcoin",
             network: "mainnet",
             derivationMode: DerivationModes.LEGACY,
-            explorer: "ledgerv3",
-            explorerURI:
-              "https://explorers.api.vault.ledger.com/blockchain/v3/btc",
+            explorerURI: `https://explorers.api.live.ledger.com/blockchain/v4/btc`,
```

Comment:
use "explorerURI: `${explorerBaseURI}/v4/btc`," as the others

Reply:
It wasn't working for some reason... i'm going to try again

Reply:
Done !